start.sh

```
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+source .env
```

Shellcheck

[warning] In POSIX sh, 'source' in place of '.' is undefined.

(SC3046)